### PR TITLE
Add GitHub templates, update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kienstra**
`CONTRIBUTING.md` has a lot of good instructions, but since it was written, it's become much more common to have issue and PR templates.

This should make it easier for us when we get bug reports, as the template has a 'Steps to reproduce' section.

Inspired by the templates in the [AMP plugin](https://github.com/ampproject/amp-wp/blob/0c7ba2b46390420ca0c6dee069d0ae5322b3084f/.github/ISSUE_TEMPLATE/bug_report.md) and [Jetpack](https://github.com/Automattic/jetpack/blob/8537cdd3fce9e3c3d0ff9847563d0ce2ea5ea8cf/.github/ISSUE_TEMPLATE/bug_report.md).

Now, when people open a bug report, it'll look something like:

new-br

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -139,7 +74,7 @@ While the prerequisites above must be satisfied prior to having your pull reques
 
 ### Local Setup
```
Comment (kienstra) [inline CONTRIBUTING.md:75]: I need to update this file a lot more, but that's outside of the scope of this PR.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+## Steps to reproduce
+
+<!-- Exactly how to reproduce it -->
+1. Navigate to...
+2. Click...
+3. Etc...
+
+## Expected behavior
+
+## Actual behaviour
```
Comment (kienstra) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Haha, I'm becoming Australian 🇦🇺

**Comment by kienstra** [comment]
**Request For Review**

Hi @lukecarbis,
No hurry here, but this PR should make it easier to read community-opened issues and PRs. 

Feel free to add anything you'd like. For example, if there's some information that you wish that all issues had.

Thanks, Luke!

**Review by lukecarbis — APPROVED** [review]
Everything here looks good to me.

**Comment by kienstra** [comment]
If it's alright, I pushed https://github.com/getblocklab/block-lab/pull/494/commits/3814e649099e9c5849b0fcb0652456532a48723d so that the `.github/` files won't be in the build deployed to wp.org.

build-here

**Pull request merged**